### PR TITLE
Updated Electron to 23.0.0 and NodeJS 18.12.0

### PR DESCRIPTION
**Pull request by rickard-von-essen**
Upgrade to ElectronJS 23.0.0 released yesterday and also bump NodeJS 18.12.0.

TODO:

- [x] Simple click around test
- [ ] Verify serial connect
- [ ] Verify...

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: package.json [hunk 1]
```diff
@@ -58,6 +58,7 @@
     "d3": "^7.8.2",
     "djv": "^2.1.4",
     "dompurify": "^2.4.0",
+    "electron": "23.0.0",
```
Comment (McGiverGim) [inline package.json:61]: I don't understand. Why we need electron?

Comment (rickard-von-essen) [reply package.json:61]: Oh! Never mind, this is just me being stupid and working on too many things at the same time BFC doesn't build on Electron. 🙈

**Comment by rickard-von-essen** [comment]
Sorry for the noise.

**Pull request closed without merging**